examples/foundry/Makefile

```diff
@@ -0,0 +1,18 @@
+
+all: build test
+
+build:
+	forge install foundry-rs/forge-std --no-commit --no-git
+	cp -r ../../contracts/contracts/ lib/contracts
+	cp -r ../../integrations/foundry/src src
```

Comment:
I think you should be able to use proper `forge install` tooling and remappings for this. Or at least use a symbolic link for `lib/contracts` and `lib/sapphire-foundry` pointing to folders above.

Reply:
Contracts are installed from updated soldeer repository https://soldeer.xyz/project/[@oasisprotocol-sapphire-contracts](https://soldeer.xyz/project/@oasisprotocol-sapphire-contracts)

Added symlink for integrations/foundry/src